content/W2016/W16-big-data-panel.md

```diff
@@ -0,0 +1,24 @@
+Title: Big Data Panel
+Date: 2016-01-18
+Category: Events
+Tags: talks, panels
+Slug: big-data-panel-w16
+Author: Anna Lorimer
+Summary: Interested in big data? Come to our Big Data Panel!
+
+Curious about Big Data or want to find out how to get involved? Several women
+who work in Big Data will be sharing their experiences and giving advice.
+Come out and hear from women working in the industry!
+
+This event is open to everyone and food will be provided.
+
+
+## Event Details ##
+
++ **Who:** [Greta Cutulenco](http://gretacutulenco.com/),
+		   [Amanda Edwards](https://ca.linkedin.com/in/amanda-edwards-18083346),
```

Comment:
Just for future reference, looks like you used tabs instead of spaces here. In general, it is good practice (with some disagreement on this topic professionally, I assure you) to expand your tabs into spaces so your code looks the same to everyone who reviews it :)